features/support/before-hook.js

```diff
@@ -30,6 +31,18 @@ function promiseToWait(timeToWait) {
 }
 
 /**
+ * Generate a random string
+ * @method randomBytes
+ * @param {Number}      size    Number of bytes to generate
+ * @return {Promise}
+ */
+function randomBytes(size) {
```

Comment:
The `crypto` package is part of node itself, so it doesn't represent any new dependency. However, the fix for your second comment about the original `lastUsed` value will lead to a cleaner solution here I think.